package.json

```diff
@@ -44,8 +44,8 @@
     "build": "lerna run build",
     "preelectron": "yarn build",
     "electron": "cd packages/fether-electron && yarn electron",
-    "lint-files": "./scripts/lint-files.sh",
-    "lint": "yarn lint-files '**/*.js'",
+    "lint-files": "./scripts/lint-files.sh '**/*.js'",
+    "lint": "yarn lint-files",
```

Comment:
Why this change? This makes the pre-commit hook less performant

Reply:
Without the change it was trying to lint the .ts files and the pre-commit hook was preventing me from pushing